src/libraries/Microsoft.PowerFx.Core/Microsoft.PowerFx.Core.csproj

```diff
@@ -21,6 +21,8 @@
 		<Description>The core library for all Microsoft Power Fx Compiler targets</Description>
 		<Summary>The core library for all Microsoft Power Fx Compiler targets</Summary>
 		<GeneratePackageOnBuild>True</GeneratePackageOnBuild>
+		<RunAnalyzersDuringBuild>true</RunAnalyzersDuringBuild>
+		<EnforceCodeStyleInBuild>true</EnforceCodeStyleInBuild>
```

Comment:
This property is in Directory.Build.props, covers every project. I don't think it needs to be repeated here.